mon-pix/app/templates/assessments/challenge.hbs

```diff
@@ -1,6 +1,6 @@
 {{page-title this.pageTitle}}
 
-<div class="background-banner-wrapper assessment-challenge">
+<main class="background-banner-wrapper assessment-challenge">
```

Comment:
J'hésite entre mettre le main ici (donc il contient le header) ou en dessous ligne 15 (pour entourer le component et donc seulement l'épreuve en elle-même, en tant qu'information principale (`main`) )

Reply:
Selon w3schools :
"The content inside the <main> element should be unique to the document. It should not contain any content that is repeated across documents such as sidebars, navigation links, copyright information, site logos, and search forms."
Je vais donc le passer ligne 15

Reply:
👍